src/main/resources/inspectionDescriptions/GenericObjectTypeUsageInspection.html

```diff
@@ -4,5 +4,7 @@
 <br />
 Refactoring this will keep code analysis simple and contribute 
 to overall better structure.
+
```

Comment:
Here we need implicit `<br/>`s, or text will be rendered as a single paragraph (same for 3 other descriptions).